configure

```diff
@@ -890,6 +897,19 @@ if (!$ace_src || !$tao_src) {
 print "Using ace_src: $ace_src\n" if $opts{'verbose'};
 print "Using tao_src: $tao_src\n" if $opts{'verbose'};
 
+if ($is_windows && $opts{'target'} eq 'android') {
+  # Rename the VERSION files in ACE, TAO, and OPENDDS. On Windows the cross
+  # compiler will pick them up instead of the version standard header the
+  # standard library is trying to include.
+  if (exists($opts{'dry-run'})) {
+    print("Would rename VERSION files to get around version header Windows issue.\n");
+  } else {
+    rename("$ace_src\\VERSION", "$ace_src\\VERSION.txt");
```

Comment:
Perl's stdlib is fine with `/` here, and we may end up using this on non-Windows platforms.  As `<version>` is in the process of becoming a C++ standard, and Windows isn't the only case-insensitive filesystem around, maybe we should go ahead and change these in the source repos.

Reply:
Yes, I'm fine with that, as long as they don't add a `configure` standard header as well...